core/src/test/java/com/pholser/junit/quickcheck/runner/PropertyFalsifiedUtilityClassTest.java

```diff
@@ -58,10 +60,10 @@ public void counterexampleFoundWithAllParametersIsCorrect() {
             seeds,
             error);
 
-        String expected = "Property named 'mySuperProperty' failed (assertion name)\n"
-            + "With arguments: [first, second, third]\n"
-            + "Seeds for reproduction: [12345, 8842]";
-        assertEquals(expected, actual.getMessage());
+        String expected = format("Property named 'mySuperProperty' failed (assertion name)%n"
+            + "With arguments: [first, second, third]%n"
+            + "Seeds for reproduction: [12345, 8842]");
```

Comment:
Personal preference: use matcher `equalTo` instead of `is`.

Reply:
Done